organize/models.py

```diff
@@ -107,9 +108,11 @@ def deploy(self):
 
         self.change_status_to(DEPLOYED)
 
-        # TODO: we should recognize here if we should create a new event,
-        # copy old one or copy old and change organizaers.
-        event = self.create_event()
+        if Event.objects.filter(city=self.city,
+                                country=self.get_country_display()).exists():
+            event = copy_event(self.previous_event, self.date)
```

Comment:
We are doing two different things here, we are checking for an event in our database and then using the previous_event from the application object. We should probably check if previous event exists OR if we have a previous event in the database, because in both cases we want to copy contents.